util/common.go

```diff
@@ -57,6 +58,17 @@ func BenchmarkResultToParseBenchmark(name string, b *testing.BenchmarkResult) *p
 	}
 }
 
+// ParseBenchmarkName removes the path and suffixes from benchmark info
+// Example: BenchmarkGoDriver/transform/accumulator_factory-4 -> accumulator_factory
+func ParseBenchmarkName(name string) string {
+	spl := strings.Split(name, "/")
+	name = spl[len(spl)-1]
```

Comment:
`filepath.Base("BenchmarkGoDriver/transform/accumulator_factory-4")`

Reply:
initially it was done using `filepath.Base` but
https://reviewable.io/reviews/bblfsh/performance/1#-Lj1YiAXCy6I0H4XMjGi

Reply:
@dennwc  - why `filepath.Base` is incorrect here? We have a absolute path here and we wanna get a benchmark file name. If we rename name param to `path` and function name to `benchamarkName` I think it will be - WDYT?